src/Pages/DashboardPage/Dashboard.css

```diff
@@ -0,0 +1,18 @@
+.dashboardcontainer{
```

Comment:
use proper camelcase. This should be `dashboardContainer`.
Use this convention in all css. Update the existing ones as well